code42-incydr-(Apps) (#5034)
SumoLogic/sumologic-documentation: docs/integrations/saas-cloud/code42-incydr.md

---
id: code42-incydr
title: Code42 Incydr
sidebar_label: Code42 Incydr

> description: The Code42 Incydr app enables security teams to monitor, detect, and respond to potential data threats across an organization, combining insightful dashboards that cover audits, file events, and user sessions.

Reviewer: ```suggestion
description: The Sumo Logic app for Code42 Incydr allows security teams to monitor, detect, and respond to potential data threats throughout an organization. It features insightful dashboards by gathering audit logs, file events, and user session data.
```

---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/send-data/code42-incydr-logo.png')} alt="Thumbnail icon" width="100"/>

> The Sumo Logic app for Code42 Incydr helps security teams monitor, detect, and respond to data threats within an organization. It provides dashboards for tracking audits, file events, and user sessions. With features to identify risky file activities and monitor user behavior, it helps prevent data leaks and intellectual property theft. Incydr also offers comprehensive analysis into user actions, risk patterns, and geolocation, giving security professionals the information needed to protect against insider threats and ensure data security and compliance standards.

Reviewer: ```suggestion
The Sumo Logic app for Code42 Incydr assists security teams in monitoring, detecting, and responding to data threats within an organization. It features dashboards that track audits, file events, and user sessions. By identifying risky file activities and monitoring user behavior, the app helps prevent data leaks and intellectual property theft. Incydr also provides a comprehensive analysis of user actions, risk patterns, and geolocation, equipping security professionals with the information necessary to safeguard against insider threats and ensure compliance with data security standards.
```

:::info

> This app includes [built-in monitors](#code42-incydr-monitors). For details on creating custom monitors, refer to the [Create monitors for Code42 Incydr app](#create-monitors-for-code42-incydr-app).

Reviewer 2: ```suggestion
This app includes [built-in monitors](#code42-incydr-monitors). For details on creating custom monitors, refer to [Create monitors for Code42 Incydr app](#create-monitors-for-code42-incydr-app).
```

> This app includes [built-in monitors](#code42-incydr-monitors). For details on creating custom monitors, refer to [Create monitors for Code42 Incydr app](#create-monitors-for-code42-incydr-app).

Reviewer: ```suggestion
This app includes [built-in monitors](#code42-incydr-monitors). For details on creating custom monitors, refer to the [Create monitors for Code42 Incydr app](#create-monitors-for-code42-incydr-app).
```

:::

## Log types

> This app uses Sumo Logic’s [Code42 Incydr Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/code42-incydr-source/) to collect audits, file events, and session logs from Code42 Incydr platform.

Reviewer: ```suggestion
This app uses Sumo Logic’s [Code42 Incydr Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/code42-incydr-source/) to collect audits, file events, and session logs from the Code42 Incydr platform.
```

| count
```

## Collection configuration and app installation

import CollectionConfiguration from '../../reuse/apps/collection-configuration.md';

<CollectionConfiguration/>

:::important
Use the [Cloud-to-Cloud Integration for Code42 Incydr](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/code42-incydr-source/) to create the source and use the same source category while installing the app. By following these steps, you can ensure that your Code42 Incydr app is properly integrated and configured to collect and analyze your Code42 Incydr data.
:::

### Create a new collector and install the app

import AppCollectionOPtion1 from '../../reuse/apps/app-collection-option-1.md';

<AppCollectionOPtion1/>

### Use an existing collector and install the app

import AppCollectionOPtion2 from '../../reuse/apps/app-collection-option-2.md';

<AppCollectionOPtion2/>

### Use an existing source and install the app

import AppCollectionOPtion3 from '../../reuse/apps/app-collection-option-3.md';

<AppCollectionOPtion3/>

## Viewing Code42 Incydr dashboards

import ViewDashboards from '../../reuse/apps/view-dashboards.md';

<ViewDashboards/>

### Audits Overview

> The **Code42 Incydr - Audits Overview** dashboard is a centralized platform that provides a snapshot of auditing activities in your organization. It tracks key metrics like total audits, trends, and success rates over time. It also highlights the most active users in the auditing process and common audit types being performed, offering insights into potential risk areas. The dashboard also segments data by user roles, search queries, and geographical locations, including embargoed areas, to show how information is accessed and monitored. It lists the top auditing tools and provides updates on recent system audits and alerts, keeping stakeholders informed about data security and compliance.

Reviewer: ```suggestion
The **Code42 Incydr - Audits Overview** dashboard provides a snapshot of auditing activities in your organization. It tracks key metrics like total audits, trends, and success rates over time. It also highlights the most active users in the auditing process and common audit types performed, offering insights into potential risk areas. The dashboard also segments data by user roles, search queries, and geographical locations, including embargoed areas, to show how information is accessed and monitored. It lists the top auditing tools and provides updates on recent system audits and alerts, keeping stakeholders informed about data security and compliance.
```

<img src={useBaseUrl('https://sumologic-app-data-v2.s3.us-east-1.amazonaws.com/dashboards/Code42-Incydr/Code42+Incydr+-+Audits+Overview.png')} alt="Audits Overview dashboard" />

### File Events

> The **Code42 Incydr - File Events** dashboard is a security command center that tracks all file events within the organization, providing detailed metrics on total and severe events, as well as specific user and file activities, enabling identification of potential data breaches and insider threats. The dashboard displays the sources of file events, including OS-specific insights, and highlights geographic risk, particularly from embargoed areas. With real-time data and detection of anomalies in file behavior, security teams can efficiently monitor, analyze, and respond to the dynamic data movement and access.

Reviewer: ```suggestion
The **Code42 Incydr - File Events** dashboard is a security command center that tracks all file events within your organization, providing detailed metrics on total and severe events, as well as specific user and file activities, enabling identification of potential data breaches and insider threats. The dashboard displays the sources of file events, including OS-specific insights, and highlights geographic risk, particularly from embargoed areas. With real-time data and detection of anomalies in file behavior, security teams can efficiently monitor, analyze, and respond to dynamic data movement and access.
```

<img src={useBaseUrl('https://sumologic-app-data-v2.s3.us-east-1.amazonaws.com/dashboards/Code42-Incydr/Code42+Incydr+-+File+Events.png')} alt="File Events dashboard" />

### Sessions

> The **Code42 Incydr - Sessions** dashboard offers a real-time view of your organization's session data and associated risk levels, with analytics on total sessions, average engagement, and a spectrum of risk events ranging from low to critical. Dynamic visualizations display session trends, geospatial engagement, and content inspections, helping security teams quickly identify and address potential data exfiltration risks. This dashboard is essential for continuous monitoring and ensuring prompt action on any anomalies or compliance issues to maintain strong data security protocols.

Reviewer: ```suggestion
The **Code42 Incydr - Sessions** dashboard offers a real-time view of your organization's session data and associated risk levels, with analytics on total sessions, average engagement, and a spectrum of risk events ranging from low to critical. Dynamic visualizations display session trends, geospatial engagement, and content inspections, helping security teams quickly identify and address potential data exfiltration risks. This dashboard is essential for continuous monitoring and ensuring prompt action on any anomalies or compliance issues to maintain strong data security protocols.
```

<img src={useBaseUrl('https://sumologic-app-data-v2.s3.us-east-1.amazonaws.com/dashboards/Code42-Incydr/Code42+Incydr+-+Sessions.png')} alt="Sessions dashboard" />

## Create monitors for Code42 Incydr app

import CreateMonitors from '../../reuse/apps/create-monitors.md';

<CreateMonitors/>

### Code42 Incydr monitors

| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition | 
|:--|:--|:--|:--|
| `Code42 Incydr - Alert Rules Disabled` | This alert is triggered when alert rules in Code42 Incydr are disabled, potentially reducing visibility into security incidents. This may indicate unauthorized changes or misconfigurations that could impact data security monitoring. | Critical | Count > 0 | 
| `Code42 Incydr - Audits from Embargoed Locations` | This alert is triggered when audit events originating from embargoed or restricted locations are detected, indicating potential compliance violations or unauthorized access attempts. | Critical | Count > 0|

> | `Code42 Incydr - File Events from Embargoed Locations` | This alert is triggered when file activity from embargoed locations, such as uploads, downloads, or modifications are identified. This alert helps detect potential data exfiltration or compliance risks. | Critical | Count > 0|

Reviewer: ```suggestion
| `Code42 Incydr - File Events from Embargoed Locations` | This alert is triggered when file activity, such as uploads, downloads, or modifications from embargoed locations are identified. This alert helps detect potential data exfiltration or compliance risks. | Critical | Count > 0|
```

> | `Code42 Incydr - Sessions with Critical Risk Events` | This alert flags user sessions associated with high-risk activities, such as large file transfers, unusual login patterns, or interactions with sensitive data, helping security teams respond to potential insider threats or compromised accounts. | Critical | Count > 0|

Reviewer: ```suggestion
| `Code42 Incydr - Sessions with Critical Risk Events` | This alert is triggered if user sessions associated with high-risk activities, such as large file transfers, unusual login patterns, or interactions with sensitive data, are detected. Helping security teams respond to potential insider threats or compromised accounts. | Critical | Count > 0|
```
